lib/msal-browser/src/request/AuthorizationCodeRequest.ts

```diff
@@ -5,7 +5,7 @@
 
 import { CommonAuthorizationCodeRequest } from "@azure/msal-common";
 
-export type AuthorizationCodeRequest = Partial<Omit<CommonAuthorizationCodeRequest, "code"|"enableSpaAuthorizationCode"|"requestedClaimsHash">> & {
+export type AuthorizationCodeRequest = Partial<Omit<CommonAuthorizationCodeRequest, "code"|"enableSpaAuthorizationCode"|"requestedClaimsHash"|"tokenQueryParameters">> & {
```

Comment:
Why is this omitted here? It should not be.

Reply:
I think I got mixed up in determining which public flows allow tokenQueryParameters. Is there a set list I can look at? Or can you tell me here?